drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -305,6 +306,23 @@ void at86rf2xx_set_csma_seed(at86rf2xx_t *dev, uint8_t entropy[2])
     at86rf2xx_reg_write(dev, AT86RF2XX_REG__CSMA_SEED_1, tmp);
 }
 
+int8_t at86rf2xx_get_cca_threshold(at86rf2xx_t *dev)
+{
+    int8_t tmp = at86rf2xx_reg_read(dev, AT86RF2XX_REG__CCA_THRES);
+    tmp &= AT86RF2XX_CCA_THRES_MASK__CCA_ED_THRES;
+    tmp <<= 1:
+    return (RSSI_BASE_VAL + tmp);
+}
+
+void at86rf2xx_set_cca_threshold(at86rf2xx_t *dev, int8_t value)
+{
```

Comment:
>  The AT86RF231 CCA threshold registers expects the actual negative value to be stored in its ad-hoc register (well, with an offset and by steps of 2).

Ok, wasn`t aware of that. The KW2x handles it the other way and expects a positive value (that represents the value in dBm: 75 -> -75dBm).

Just wanted to point out that, in my opinion, it shouldn`t make a difference if we insert positive or negative values in the shell or somewhere else. Real positive values (e.g. +10dBm) are not a realistic scenario, so they can be mapped to -10dBm to avoid confusion. Do you agree?